camelot/admin/action/list_filter.py

```diff
@@ -271,12 +272,79 @@ def get_state(self, model_context):
         state.modes = modes
         return state
 
-
+class SearchFieldStrategy(object):
+    """Abstract class for search field strategies.
+       It offers an interface for defining a search clause for a given column and search text.
+    """
+        
+    @classmethod
+    def get_clause(cls, column, text):
+        """Return a search clause for the given column and search text, if applicable."""
+        raise NotImplementedError
+  
+class BasicSearch(SearchFieldStrategy):
+    
+    @classmethod
+    def get_clause(cls, c, text):

```

Comment:
nota aan onszelf : we zouden deze bedenkelijke if subclass afhaspeling kunnen vermijden door de search field attribute mee te zetten in camelot/view/field_attributes.py

en in de ObjectAdmin.get_field_attributes default op NoSearch te zetten of zo

Reply:
Ok, zie 4f4405c25717c549b3b30ade23752e90e48ef2f8

Reply:
Ok, zie  #56